action.yml

```diff
@@ -253,7 +262,7 @@ runs:
         echo "Infra state is: $infra_error"
         echo "::set-output name=FINAL_STATE::$new_state"
         echo "::set-output name=INFRA_STATE::$infra_error"
-        echo "RESULTS_URL=$tf_url" >> $GITHUB_ENV
+        echo "::set-output name=RESULT_URL::$tf_url
```

Comment:
What if we do not parse `result_url` at all, but use the same value as in `request_url` output, so that they are consistent?

Reply:
The best way to add into all our container-tests.yml files `tf_scope` to `public` or `private` and we can use `result_url` in all our containers. Like OAMG team does it.

Reply:
I meant, that the `results_url` displayed in the PR status should be the same, as is the `request_url` in outputs:
 82   request_url:
 83     description: 'An url of a scheduled testing farm request'
 84     value: ${{ steps.sched_test.outputs.results_url }}
 
 and not parsed from the JSON, TF sends us.
 But it is possible, I read the situation incorrectly.